mysql/resource_user.go

```diff
@@ -59,13 +60,18 @@ func resourceUser() *schema.Resource {
 				ConflictsWith:    []string{"plaintext_password", "password"},
 			},
 
+			"aad_identity": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				ForceNew:         true,
+				DiffSuppressFunc: NewEmptyStringSuppressFunc,
+			},
+
 			"auth_string_hashed": {
 				Type:             schema.TypeString,
 				Optional:         true,
 				Sensitive:        true,
 				DiffSuppressFunc: NewEmptyStringSuppressFunc,
-				RequiredWith:     []string{"auth_plugin"},
-				ConflictsWith:    []string{"plaintext_password", "password"},
```

Comment:
Is there any reason to remove ConflictsWith?

Reply:
No, this is mistake, sending fix